src/sonic-config-engine/tests/test_minigraph_case.py

```diff
@@ -0,0 +1,111 @@
+from unittest import TestCase
```

Comment:
I don't think we need to maintain a full copy of test cases in this file. This is just a subset of main test file and is meant to test this issue only. I keep several test cases instead of only one because that this issue could affect parsing in different sections.